deephyper/skopt/optimizer/optimizer.py

```diff
@@ -651,10 +673,12 @@ def _filter_failures(self, yi):
         if self.filter_failures in ["mean", "max"]:
             yi_no_failure = [v for v in yi if v != OBJECTIVE_VALUE_FAILURE]
 
-            if self.filter_failures == "mean":
-                yi_failed_value = np.mean(yi_no_failure)
+            if not yi_no_failure:
+                yi_failed_value = np.nan
+            elif self.filter_failures == "mean":
+                yi_failed_value = np.mean(yi_no_failure, axis=0).tolist()
```

Comment:
similarly to the constant liar behaviour in the MO case we need to document the behaviour to replace failures in the MO case mean/min of each objective